download.py

```diff
@@ -846,9 +846,14 @@ def download(asset_data, **kwargs):
         asset_data = copy.deepcopy(asset_data)
     else:
         asset_data = asset_data.to_dict()
+
+    #inject resolution into prefs.
+    PREFS = utils.get_preferences_as_dict()
```

Comment:
Lets not use capitals for non-constants.

Reply:
solved